src/js/rebuses.js

```diff
@@ -902,7 +902,12 @@ const rebuses = [
     symbols: ['🦀', '+', '🎂'],
     words: ['crab', 'cake'],
     hint: 'A delicious appetizer'
-  }
+  },
+  {
+    symbols: ['👃🏾', '+', ' 💍'],
+    words: ['nosering'],
+    hint: 'the jewellry for nose piercing'
+  },
```

Comment:
Pretty sure the failing build thing is because of the comma on line 910. I think if you remove it, your PR should pass.